contrib/ohi-release-notes/action.yml

```
@@ -65,11 +62,19 @@ runs:
       shell: bash
       run: |
         echo "::set-output name=skip::${{ steps.empty.outputs.is-empty == 'true' || steps.held.outputs.is-held == 'true' }}"
+    # rt-dictionary is a file contained by this action and is located inside $GITHUB_ACTION_PATH, but this
+    # folder is not mounted in the container that runs the link-dependencies action, so we copied the file to
+    # /home/runner/work/_temp/_github_home/ which is mounted in /github/home/
+    - name: Copy dictionary to temp folder mounted by the link-dependencies action
+      shell: bash
+      run: |
+        cp ${{ github.action_path }}/rt-dictionary.yml /home/runner/work/_temp/_github_home/rt-dictionary.yml
```

I don't like to have these kinds of absolute paths that might not be standard because they can be changed by Github without notice as they seem to be internal.

Take a look at this issue from a year ago: https://github.com/actions/checkout/issues/785#issuecomment-1133397851
Runners were run in `/opt/actions-runner/_work/`

If the image that is built for this step only has access to `${GITHUB_WORKSPACE}`, let's use it.

```suggestion
        if ! [ -f "${GITHUB_WORKSPACE}/${{ inputs.link-dependencies-dictionary }}" ]; then
                cp ${{ github.action_path }}/rt-dictionary.yml "${GITHUB_WORKSPACE}/${{ inputs.link-dependencies-dictionary }}"
        fi
```

This way, we have a default for users to override our dictionary and we set a default in case it does not exist for us.

---

sounds good, I will do a quick check to see if that works

---

thanks! [18c1d36](https://github.com/newrelic/release-toolkit/pull/152/commits/18c1d36223c4664ded6407f4c027b892c1c4d00b)